driverkit/config/dev/flatcar_stable_4.19.106-flatcar.yaml

```diff
@@ -0,0 +1,5 @@
+kernelrelease: 4.19.106-flatcar
+target: flatcar-stable
```

Comment:
Hi @mazzy89 
I believe we need [this](https://github.com/falcosecurity/driverkit/issues/21#issuecomment-601117055) done before. 
Alternatively, if the configuration works just with `target: vanilla`, we could use it now, and later we will add the flatcar target as further optimization.
Let me know if you need any help with that!

Reply:
Hi @leogr thank you for the input here.
I forgot to ask last time. how can we make sure that the `target: vanilla` used to build the driver for flatcar actually work?

Reply:
Firstly, we should make sure it works by [manual installation](https://github.com/falcosecurity/driverkit/issues/23). Have you tried?

If so - and if I am not wrong - it should also work on all kernels with the same `kernelconfigdata`.

Reply:
This test for me then would require a bit more of time because being in a cluster i would need to change the userdataa and then make an entire rollout